code/menuui/mainhallmenu.cpp

```diff
@@ -882,12 +885,9 @@ void main_hall_do(float frametime)
 				case MULTIPLAYER_REGION:
 					// Make sure we are in multi mode.
 					Player->flags |= PLAYER_FLAGS_IS_MULTI;
```

Comment:
Should this flag also be moved?

Alternatively, is this flag properly cleared if there is an error?

Reply:
Pretty sure this flag is set multiple times before you even get here. It doesn't affect multi state in general, just indicates that the current pilot was last used in multi. Because pilots aren't separated into single/multi any longer this flag is used to say whether we are in single or multi mode (for player select, barracks, etc.)

Reply:
It's also good to have this here in case you start a multi game from a mainhall when the pilot in in SP mode.